src/ig/IGExporter.ts

```diff
@@ -81,50 +90,33 @@ export class IGExporter {
   }
 
   /**
-   * Initializes the ImplementationGuide JSON w/ data from the package.json
+   * Initializes the ImplementationGuide JSON w/ data from the configuration YAML
    *
    * @see {@link https://confluence.hl7.org/pages/viewpage.action?pageId=35718629#NPMPackageSpecification-PackageManifestpropertiesforIGs}
    */
-  private initIG(): void {
-    const packageJSON = this.pkg.packageJSON;
-    // TODO: Consider adding a generated file warning when the IG Publisher supports JSON5-style comments
+  private initIG() {
+    // first, properties that can be directly used without much trouble
     this.ig = {
       resourceType: 'ImplementationGuide',
-      id: packageJSON.name,
-      url: `${packageJSON.canonical}/ImplementationGuide/${packageJSON.name}`,
-      version: packageJSON.version,
+      id: this.config.id,
+      meta: this.config.meta,
+      implicitRules: this.config.implicitRules,
+      language: this.config.language,
+      text: this.config.text,
+      contained: this.config.contained,
+      modifierExtension: this.config.modifierExtension,
+      url: this.config.url ?? `${this.config.canonical}/ImplementationGuide/${this.config.id}`,
+      version: this.config.version,
       // name must be alphanumeric (allowing underscore as well)
-      name: (packageJSON.title ?? packageJSON.name).replace(/[^A-Za-z0-9_]/g, ''),
-      title: packageJSON.title ?? packageJSON.name,
-      status: 'active', // TODO: make user-configurable
-      publisher: packageJSON.author,
-      contact: packageJSON.maintainers?.map(m => {
-        const contact: ContactDetail = {};
-        if (m.name) {
-          contact.name = m.name;
-        }
-        if (m.url) {
-          contact.telecom = [
-            {
-              system: 'url',
-              value: m.url
-            }
-          ];
-        }
-        if (m.email) {
-          contact.telecom = contact.telecom ?? [];
-          contact.telecom.push({
-            system: 'email',
-            value: m.email
-          });
-        }
-        return contact;
-      }),
-      description: packageJSON.description,
-      packageId: packageJSON.name,
-      license: packageJSON.license,
-      fhirVersion: ['4.0.1'],
-      dependsOn: [],
+      name: this.config.name.replace(/[^A-Za-z0-9_]/g, ''),
+      title: this.config.title,
+      status: this.config.status,
+      publisher: this.config.publisher,
```

Comment:
I don't see where `experimental` and `date` are handled. I think they are simple copy overs and would go here?